dallinger/recruiters.py

```diff
@@ -215,7 +219,10 @@ def notify_recruited(self, participant):
 
     def reward_bonus(self, assignment_id, amount, reason):
         """Reward the Turker for a specified assignment with a bonus."""
-        return self.mturkservice.grant_bonus(assignment_id, amount, reason)
+        try:
+            return self.mturkservice.grant_bonus(assignment_id, amount, reason)
+        except MTurkServiceException as ex:
+            logger.exception((ex.message))
```

Comment:
Are the nested parens needed here?

Reply:
No. That's my editor jumping the gun on me. Thanks; will fix!